org.adempiere.base/OSGI-INF/org.adempiere.base.AnnotationBasedModelFactory.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.3.0" activate="activate" name="org.adempiere.base.AnnotationBasedModelFactory">
+   <service scope="bundle">
+      <provide interface="org.adempiere.base.IModelFactory"/>
+   </service>
+   <implementation class="org.adempiere.base.AnnotationBasedModelFactory"/>
+</scr:component>
```

Comment:
You have duplicate component definition here. I believe one is manually created and the other auto generated from "@Component" annotation.

Reply:
Good catch. Fixed.